### PR TITLE
Add specification regarding proxy usage

### PR DESCRIPTION
**Pull request by jefferson-1**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -62,6 +63,25 @@ ethminer --help
 
 Check our [samples](POOLEXSAMPLES_ETH.md) to see how to connect to different pools.
 
+### Examples connecting proxy
+
+Ethminer supports using a proxy server, such as eth-proxy: https://github.com/Atrides/eth-proxy
+
+If multiple miners are running ethminer they can all connect to a single proxy. This increases the efficiency of the mining operation.
```
Comment (AndreaLanfranchi) [inline README.md]: The statement "This increases efficiency of the mining operation" is false and misleading.

**Comment by smurfy** [comment]
i too think its misleading, also eth-proxy is outdated and only in edge cases maybe faster.

In my opinion its only benefit is combining multiple worker as "one" worker to the pool

**Comment by AndreaLanfranchi** [comment]
> In my opinion its only benefit is combining multiple worker as "one" worker to the pool

Even in that case it's efficiency is arguable. Eth-proxy was born as a _bridge_ get_work -> stratum for miners that were still not supporting stratum directly. There is no meaning in loosing the benefits of pushed works (stratum) forcing ethminer with very low intervals of --farm-recheck ... I've personally tested the number of stale shares generated with eth-proxy is always way higher than direct stratum connections.

**Comment by jefferson-1** [comment]
Given the negative reviews on this I'll close this pull request.

**Comment by StefanOberhumer** [comment]
**Just as info:**

As I saw this PR, I tried eth-proxy for about 16 hours hoping ```... increases the efficiency of the mining operation. ```

The disadvantage of the proxy: :-1: 
* I got 1,89% more stale shares 

The benefit of the proxy: :+1: 
* you can easily change the wallet or pool without changing/restarting/rebooting you miners.
* if you have a really bad internet connection (WIFI or something) possible eth-proxy could  be faster.

As I have stopped hopping pools and have a reasonable internet connection I went back to (direct) stratum connection.

**Comment by jefferson-1** [comment]
As it happens, I do have some miners on some wifi connections that are 3mbps down, 1mbps up. They are on a proxy because it does perform better than direct connections.

**Comment by AndreaLanfranchi** [comment]
@StefanOberhumer @jefferson-1 

Problem of eth-proxy is that works have to be pulled by ethminer which gives a lot of overhead as ethminer itself keeps sending get_work method over http. And as you lower --farm-recheck value the more overhead you create.

On the other hand, with stratum, it's the pool which "pushes" works to miner's listening socket whenever they're available. So there is no overhead.

I personally do not believe poor internet connection are the problem. I believe instead that poor routers and firewalls may be the bottleneck as the more sockets you keep opened the more those device fatigue to keep up. 1Mb bandwith in upload is more than enough for the data stream but may cause lags if the latency is high.

Only valid reason I could find to use a proxy is if you want to "cluster" several rigs into a single virtual machine. Here comes in play a stratum/stratum proxy which holds a single connection to pool and delivers the same jobs to all connected rigs: in such case, as ethminer pseudo-guarantees each GPU has a search nonce range which does not overlap with other GPUs, you can maximize your efficiency per work.

I've written one in .net which works quite well right now but is only compatible with stratum (no eth-proxy and no nicehash mode) that I use proficiently with ethminer.

If you're interested ...

**Pull request closed without merging**